test/integration/case-management.v1.test.js

```diff
@@ -0,0 +1,487 @@
+/* eslint-disable no-console */
+/**
+ * (C) Copyright IBM Corp. 2020.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+'use strict';
+
+const CaseManagementV1 = require('../../dist/case-management/v1');
+const authHelper = require('../resources/auth-helper.js');
+
+// testcase timeout value (60s).
```

Comment:
Minor but the comment needs to be 120s or the timeout value should be 60000.

Reply:
👍